js/views/SurveyListPage.js

```diff
@@ -35,11 +37,22 @@ const SurveyListPage = React.createClass({
   },
 
   componentDidMount() {
-    this._surveys = loadCachedSurveyList().slice();
-    if (this._surveys.length === 0) {
-      loadSurveyList(this.loadList);
+    NetInfo.fetch().done((reach) => {
```

Comment:
That is a good point. I'll update this to ignore those network calls, but not ignore those trigger checks.